extension/src/extension.ts

```diff
@@ -28,12 +28,13 @@ export function activate(context: ExtensionContext) {
 		// Register the server for plain text documents
 		documentSelector: ["haskell"]
 	};
-	let client2 = new LanguageClient( 
+	client = new LanguageClient(
 		'haskell',
 		'Haskell IDE Core',
```

Comment:
Shouldn't this be ghcide?

Reply:
good catch, that’s leftover from before the renaming.